From 69b3dbbb2c85e2be612e38b88ef9867d910315cd Mon Sep 17 00:00:00 2001
From: Agriya Khetarpal <74401230+agriyakhetarpal@users.noreply.github.com>
Date: Thu, 15 Aug 2024 17:00:00 +0530
Subject: [PATCH] Set `PYBIND11_FINDPYTHON` `ON` for `FindPython`

Enables `PYBIND11_FINDPYTHON` for CMake's `FindPython` module to find the
Python interpreter and libraries.

This can be removed when incorporating the next release for iminuit
which will have this enabled by default.

Co-Authored-By: Henry Schreiner <HenrySchreinerIII@gmail.com>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b057ebe..bce4f2e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,7 @@ set(CMAKE_CXX_STANDARD
 set(CMAKE_CXX_STANDARD_REQUIRED ON) # optional, ensure standard is supported
 set(CMAKE_CXX_EXTENSIONS OFF) # optional, keep compiler extensionsn off
 
+set(PYBIND11_FINDPYTHON ON)
 find_package(pybind11 CONFIG REQUIRED)
 
 file(GLOB SOURCES_A "src/*.cpp")
-- 
2.39.3 (Apple Git-146)
